charts/backingservices/charts/srs/templates/_helpers.tpl

```diff
@@ -125,6 +125,45 @@ app.kubernetes.io/instance: {{ .Release.Name }}
 {{- end }}
 {{- end }}
 
+{{- define "elasticsearch.authProvider" -}}
+{{- if (.Values.srsStorage.provisionInternalESCluster) -}}
+{{- "basic-authentication" }}
+    {{- else }}
```

Comment:
Can we have this else block inline to `if` to get a clear understanding of code.